packages/css/src/identifier.ts

```diff
@@ -3,23 +3,46 @@ import hash from '@emotion/hash';
 import { getIdentOption } from './adapter';
 import { getAndIncrementRefCounter, getFileScope } from './fileScope';
 
-function getDevPrefix(debugId: string | undefined) {
+function getDevPrefix({
+  debugId,
+  debugFileName,
+}: {
+  debugId?: string;
+  debugFileName: boolean;
+}) {
   const parts = debugId ? [debugId.replace(/\s/g, '_')] : [];
-  const { filePath } = getFileScope();
 
-  const matches = filePath.match(
-    /(?<dir>[^\/\\]*)?[\/\\]?(?<file>[^\/\\]*)\.css\.(ts|js|tsx|jsx)$/,
-  );
+  if (debugFileName) {
+    const { filePath } = getFileScope();
+
+    const matches = filePath.match(
+      /(?<dir>[^\/\\]*)?[\/\\]?(?<file>[^\/\\]*)\.css\.(ts|js|tsx|jsx)$/,
```

Comment:
We hit and issues with this the other day where it didn't match on `.cjs` and `.mjs`. Could we add those while you're here?

Reply:
Added.